crates/voicevox_core_java_api/src/common.rs

```diff
@@ -0,0 +1,107 @@
+use anyhow::Result;
+use jni::JNIEnv;
+use once_cell::sync::Lazy;
+use tokio::runtime::Runtime;
+
+pub static RUNTIME: Lazy<Runtime> = Lazy::new(|| {
+    if cfg!(target_os = "android") {
+        android_logger::init_once(
+            android_logger::Config::default()
+                .with_tag("VoicevoxCore")
```

Comment:
全然何でも良さそうですけど、統一されてないのも若干もやっとするので、こうしときます･･･？ 😇 
```suggestion
                .with_tag("voicevoxcore")
```

Reply:
いや別にいっか！　すいません撤回します！